.github/workflows/ci.yml

```
@@ -0,0 +1,61 @@
+# Environment variables
+.env
```

actionlint (1.7.11)

[error] "jobs" section is missing in workflow

(syntax-check)